AdobeIms/etc/di.xml

```diff
@@ -11,6 +11,12 @@
     <preference for="Magento\AdobeImsApi\Api\Data\ConfigInterface" type="Magento\AdobeIms\Model\Config"/>
     <preference for="Magento\AdobeImsApi\Api\Data\TokenResponseInterface" type="Magento\AdobeIms\Model\OAuth\TokenResponse"/>
     <preference for="Magento\AdobeImsApi\Api\GetTokenInterface" type="Magento\AdobeIms\Model\GetToken"/>
+    <preference for="Magento\AdobeImsApi\Api\GetImageInterface" type="Magento\AdobeIms\Model\GetImage"/>
     <preference for="Magento\AdobeImsApi\Api\UserAuthorizedInterface" type="Magento\AdobeIms\Model\UserAuthorized"/>
     <preference for="Magento\AdobeImsApi\Api\LogOutInterface" type="Magento\AdobeIms\Model\LogOut"/>
+    <type name="Magento\AdobeIms\Model\GetImage">
+        <arguments>
+            <argument name="defaultImage" xsi:type="string">https://a5.behance.net/27000444e0c8b62c56deff3fc491e1a92d07f0cb/img/profile/no-image-276.png</argument>
```

Comment:
It's better to move this to config.xml and ConfigInterface

Reply:
ok